falcon/request.py

```diff
@@ -1021,12 +829,14 @@ def _parse_form_urlencoded(self):
                            'will be ignored.')
 
         if body:
-            extra_params = parse_query_string(
+            form_params = uri.parse_query_string(
                 body,
                 keep_blank_qs_values=self.options.keep_blank_qs_values,
             )
 
-            self._params.update(extra_params)
```

Comment:
Since `self._params` no longer contains form params, we will need to modify `req.params` to avoid a breaking change.